caravel/tasks.py

```diff
@@ -0,0 +1,160 @@
+import celery
+from caravel import models, app, utils
+from sqlalchemy import create_engine, select, text
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.sql.expression import TextAsFrom
+import sqlparse
+import pandas as pd
+import time
+
+celery_app = celery.Celery(config_source=app.config.get('CELERY_CONFIG'))
+
+
+def init_query(database_id, sql, user_id, milis):
+    """Initializes the models.Query object.
+
+    :param database_id: integer
+    :param sql: sql query that will be executed
+    :param user_id: integer
+    :param milis: integer, unit time in miliseconds
+    :return: models.Query
+    """
+
+    query = models.Query()
+    query.user_id = user_id
+    query.database_id = database_id
+    if app.config.get('SQL_MAX_ROW'):
+        query.query_limit = app.config.get('SQL_MAX_ROW')
+    query.query_name = str(milis)
```

Comment:
`str` isn't in the main namespace in py3. I usually go with `'{}'.format(obj)` instead nowadays